qubes/vm/qubesvm.py

```diff
@@ -1359,6 +1362,10 @@ async def _kill_locked(self):
             raise
 
         # make sure all shutdown tasks are completed
+        try:
+            await asyncio.wait_for(self.__waiter, timeout=10_000_000)
```

Comment:
Timeout is in seconds, so this is way too long.

Reply:
oops… I thought it was in microseconds :woman_facepalming: